web/src/components/storage/dasd/DASDTable.test.tsx

```diff
@@ -27,19 +27,40 @@ import DASDTable from "./DASDTable";
 
 let mockDASDDevices: DASDDevice[] = [];
 
+let consoleErrorSpy;
 jest.mock("~/queries/storage/dasd", () => ({
   useDASDDevices: () => mockDASDDevices,
   useDASDMutation: () => jest.fn(),
   useFormatDASDMutation: () => jest.fn(),
 }));
 
 describe("DASDTable", () => {
+  beforeAll(() => {
+    consoleErrorSpy = jest.spyOn(console, "error");
+    consoleErrorSpy.mockImplementation();
+  });
+
+  afterAll(() => {
+    consoleErrorSpy.mockRestore();
+  });
```

Comment:
If I'm not wrong, you don't need this mocking here. Do you?

I guess you have added it because the console.error you got in the test output. But this piece of code, IMHO are just hiding useful information that we should address at some point. 

If I'm right, please drop these mocks.